app/assets/javascripts/pieces.js.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
Do we use coffescript? If not, you can delete this file and leave pieces.js if needed